components/eamxx/CMakeLists.txt

```diff
@@ -197,7 +197,7 @@ set(NetCDF_Fortran_PATH ${DEFAULT_NetCDF_Fortran_PATH} CACHE FILEPATH "Path to n
 set(NetCDF_C_PATH ${DEFAULT_NetCDF_C_PATH} CACHE FILEPATH "Path to netcdf C installation")
 set(SCREAM_MACHINE ${DEFAULT_SCREAM_MACHINE} CACHE STRING "The CIME/SCREAM name for the current machine")
 option(SCREAM_MPI_ON_DEVICE "Whether to use device pointers for MPI calls" ON)
-option(SCREAM_ENABLE_MAM "Whether to enable MAM aerosol support" OFF)
+option(SCREAM_ENABLE_MAM "Whether to enable MAM aerosol support" ON)
```

Comment:
Yes, you are; MAM4xx will enable after this PR. MAM4xx does not affect production runs. The only way of adding MAM4xx will be via the input file.  In addition, as the AT demonstrates, MAM4xx/optics is not affecting the outputs of the current tests, because baselines tests are passing.

Reply:
MAM4xx will only increase 8% the compilation and run time of emaxx in weaver; see this section in the PR text: **Compilation and run wall-times.**